### DISCUSSION TITLE
How do you actually forecast a time series?

### DISCUSSION THREAD

**Discussion by alfin0831** [timeseriesAI/tsai#472] [Q&A]
I'm sorry I'm really new to this AI stuff, but my question is how do you actually predict future values?

I am stuck where it says:

raw_preds, target, preds = fcst.get_X_preds(X[splits[0]], y[splits[0]])

I know that target are the values you are trying to predict and preds are the values predicted. But preds is based on the past predictions. My question is what's the next step in predicting the future values?

**Comment by oguiza** [comment]
Hi @alfin0831,
For future values you just have the X. So you do this: 
````python
raw_preds, _, decoded_preds = fcst.get_X_preds(X_to_be_pred)
```

**Reply by alfin0831** [reply]
Thank you so much sir. Have a great day!
